Sources/SwiftProtobuf/Visitor.swift

```diff
@@ -445,8 +448,33 @@ public protocol Visitor {
   mutating func visitUnknown(bytes: Data) throws
 }
 
+/// Provides options for how visitor traversal should be carried out
+public struct TraversalOptions {
+  static let `default` = TraversalOptions()
```

Comment:
@tbkka (and others) should we do this in all the different `*Options` classes?

I think we mostly just have the default values correct so an empty initializer works, but maybe this makes sense also? And should it be `public`?

Reply:
IMO, this is redundant with the no-argument constructor. A `default` static property makes more sense in class-based APIs like `FileManager` where it represents an instance that's lazily created and cached for future invocations. This is just a simple value construction, so it doesn't need it.

Reply:
Good point about it not being necessary for value types. I wasn't sure if that was the case. I'll remove.